### PR TITLE
Release/9.1

### PR DESCRIPTION
**Pull request by flomnes**

### CODE REVIEW

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -7,7 +7,7 @@ set(ANTARES_VERSION_REVISION  0)
 
 # Beta release
 set(ANTARES_BETA 0)
-set(ANTARES_RC 0)
+set(ANTARES_RC 4)
```
Comment (flomnes) [inline src/CMakeLists.txt]:
```suggestion
set(ANTARES_RC 0)
```

**Review by flomnes — COMMENTED** [review]
@payetvin Merge all 9.1.0-rc* into just 9.1.0

File: docs/developer-guide/CHANGELOG.md [hunk 2]
```diff
@@ -4,13 +4,65 @@ toc_depth: 2
...
-### Improvements
+## New features
+* Scenarized & hourly values for hydro pumping and hydro generation. Previously this data was not scenarized and daily.
 * STS groups are now "dynamic" : group names are no longer fixed by code, user is free to define these groups.
 * When XPRESS or SCIP solver is selected (using OR-Tools), solver specific parameters can be specified from the command-line (#1837)
 
-## 9.0.0
+## Improvements
+* Remove sc-builder prefix "hgp", use "h" instead for max hydro pumping & generation timeseries.
+* Add optimization options in OR-Tools / XPRESS (#1837)
```
Comment (flomnes) [inline docs/developer-guide/CHANGELOG.md]: Feature

File: docs/developer-guide/CHANGELOG.md [hunk 3]
```diff
@@ -4,13 +4,65 @@ toc_depth: 2
...
+* Rationalize consistency checks on the number of columns (#2073)
+* Time Series generation refactor (#2045)
+* Create & forward declare BasisStatus struct, reduce build time (#2044)
```
Comment (flomnes) [inline docs/developer-guide/CHANGELOG.md]: For devs

**Review by payetvin — APPROVED** [review]

**Pull request merged**